CI/build-windows.ps1

```diff
@@ -0,0 +1,142 @@
+Param(
+    [Switch]$Help,
+    [Switch]$Quiet,
+    [Switch]$Verbose,
+    [Switch]$NoChoco,
+    [Switch]$Package,
+    [Switch]$SkipDependencyChecks,
+    [Switch]$BuildInstaller,
+    [Switch]$CombinedArchs,
+    [String]$BuildDirectory = "build",
+    [String]$BuildArch = (Get-CimInstance CIM_OperatingSystem).OSArchitecture,
+    [String]$BuildConfiguration = "RelWithDebInfo"
+)
+
+##############################################################################
+# Windows OBS build script
+##############################################################################
+#
+# This script contains all steps necessary to:
+#
+#   * Build OBS with all required dependencies
+#   * Create 64-bit and 32-bit variants
+#
+# Parameters:
+#   -Help                   : Print usage help
+#   -Quiet                  : Supress most build process output
+#   -Verbose                : Enable more verbose build process output
+#   -SkipDependencyChecks   : Skips dependency checks
+#   -NoChco                 : Skip automatic dependency installation
+#                             via Chocolatey
+#   -BuildDirectory         : Directory to use for builds
+#                             Default: Win64 on 64-bit systems
+#                                      Win32 on 32-bit systems
+#  -BuildArch               : Build architecture to use (32bit or 64bit)
+#  -BuildConfiguration      : Build configuration to use
+#                             Default: RelWithDebInfo
+#  -BuildInstaller          : Build InnoSetup installer - Default: off"
```

Comment:
This isn't mentioned below in `Print-Usage`, and we don't use InnoSetup for our installer.

Reply:
Removed. Was an artifact of the plugin template where InnoSetup is indeed used.